index.d.ts

```diff
@@ -1486,13 +1486,17 @@ declare namespace Shopify {
 
   interface IFulfillmentService {
     callback_url: string;
+    email: string | null;
     format: 'json';
     handle: string;
+    id: number;
+    include_pending_stock: boolean;
     inventory_management: boolean;
     location_id: number;
     name: string;
     provider_id: number | null;
     requires_shipping_method: boolean;
+    service_name: string | null;
```

Comment:
Can this be `null`?

Reply:
I'm not sure, but because it's officially undocumented in the table of properties, I did not want to set it as a hard requirement. 

Based on the numerous API calls i've made to shopify, I think this is probably always only going to be a `string`, but I don't know that 100% for sure. I would have no arguments if you think we should remove `null` here.